### PR TITLE
first submission

### PR DESCRIPTION
**Pull request by mohit-90**
Missing:
1. the slider in the header
2. horizontal align the footer text

### CODE REVIEW

File: ex1-files/ex1.html [hunk 1]
```diff
@@ -1 +1,82 @@
...
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	</head>
+
+	<body>
+		<div class="body">
```
Comment (talmog) [inline ex1-files/ex1.html]: strange class name.  It may be confusing. We may confuse it with `<body>`

File: ex1-files/ex1.html [hunk 2]
```diff
@@ -1 +1,82 @@
...
+
+			<div class="navigation">
```
Comment (talmog) [inline ex1-files/ex1.html]: you can use the html element `<nav>`

File: ex1-files/ex1.html [hunk 3]
```diff
@@ -1 +1,82 @@
...
+				<div class="logo"><a href="#logo">SPECCTR MAG</a></div>
+				<div class="items">
+					<a href="#">Home</a>
+			  	<a href="#">About</a>
```
Comment (talmog) [inline ex1-files/ex1.html]: indentation

File: ex1-files/ex1.html [hunk 4]
```diff
@@ -1 +1,82 @@
...
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	</head>
+
+	<body>
+		<div class="body">
+
+			<div class="navigation">
+				<div class="logo"><a href="#logo">SPECCTR MAG</a></div>
+				<div class="items">
```
Comment (talmog) [inline ex1-files/ex1.html]: items is not a descriptive class name

File: ex1-files/ex1.html [hunk 5]
```diff
@@ -1 +1,82 @@
...
+							<button class="button">SIGN UP NOW</button>
+					</div>
+				</div>
+
+				<div class="text">This day is design history</div>
+
+			</div>
+
+
+			<div class="footer">
```
Comment (talmog) [inline ex1-files/ex1.html]: you can use the html `<footer`> Element

File: ex1-files/ex1.html [hunk 6]
```diff
@@ -1 +1,82 @@
...
+				<div class="copyright">
+					<span>&#169; 2011 On Pixel, Inc. - License Agreeement - Contact Us - Back to top</span>
```
Comment (talmog) [inline ex1-files/ex1.html]: `&copy;` is more readable than `&#169;`

**Review by talmog — CHANGES_REQUESTED** [review]
For the css, let's meet and go over this, we need to restructure some things.

**Comment by mohit-90** [comment]
@tomeralmog this is ready to review again. Please review.

File: ex1-files/index.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    <a href="#">Events</a>
+    <a href="#">News</a>
+    <a href="#">Contact</a>
+  </div>
+</nav>
+
+<div class="blueprint">
+  <div class="content">
+    <div class="word">
+      <span style="background-color: white">BLUE-</span>
```
Comment (talmog) [inline ex1-files/index.html]: don't use inline styles. They are not reusable and overwrite any other styles.

File: ex1-files/index.html [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+      <br/>
+      <span style="background-color: white">PRINTS</span>
```
Comment (talmog) [inline ex1-files/index.html]: no inline styles

File: ex1-files/index.html [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+    </div>
+  </div>
+  
+  <div class="text">This day is design history</div>
+</div>
+
+
+<div class="footer">
+  <div class="copyright">
+    <span style="padding-left: 36px">&copy; 2011 On Pixel, Inc. - License Agreeement - Contact Us - Back to top</span>
```
Comment (talmog) [inline ex1-files/index.html]: no inline styles

File: ex1-files/style.css [hunk 10]
```diff
@@ -0,0 +1,232 @@
+body {
+  margin: 0%;
```
Comment (talmog) [inline ex1-files/style.css]: `margin: 0`

File: ex1-files/style.css [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+
+.header-actions a:hover {
+  text-decoration: underline;
+  border-bottom: solid 4px black;
+  text-align: center;
+}
+
+.items a {
+  font-size: 14px;
+  color: rgb(0,0,0);
```
Comment (talmog) [inline ex1-files/style.css]: `color: black` or `color: #000` is more readable

File: ex1-files/style.css [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (talmog) [inline ex1-files/style.css]: I would set a font family for the body for the entire site.

File: ex1-files/style.css [hunk 13]
```diff
@@ -0,0 +1,232 @@
...
+  text-align: center;
+}
+
+.blueprint {
+  background-image: url("../ex1-files/sample1.jpg");
+  background-size: cover;
+  background-position: bottom center;
```
Comment (talmog) [inline ex1-files/style.css]: you can combine the background image and background position:
`background: url("../ex1-files/sample1.jpg") bottom center;`

File: ex1-files/style.css [hunk 14]
```diff
@@ -0,0 +1,232 @@
...
+  height: 700px;
+  width: 100%;
+  border-bottom: solid 8px black;
+}
+
+.blueprint .content .word {
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (talmog) [inline ex1-files/style.css]: same as above

File: ex1-files/style.css [hunk 15]
```diff
@@ -0,0 +1,232 @@
...
+.linkbox {
+  padding-top: 10px;
+  padding-left: 36px;
+  width: 50%;
+  box-sizing: border-box;
+}
+
+.blueprint .content .link {
+  position: relative;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (talmog) [inline ex1-files/style.css]: same as above

**Review by talmog — CHANGES_REQUESTED** [review]
Almost there.
Make sure you define the font family in one place

File: ex1-files/style.css (hunk 13 above)
Comment (mohit-90) [reply ex1-files/style.css]: yeah, thanks.

**Comment by mohit-90** [comment]
@tomeralmog updated as per comments, please review. Thanks.

**Comment by mohit-90** [comment]
sry, I committed from my other account

**Pull request merged**